internal/server/handlers.go

```diff
@@ -565,9 +565,8 @@ func (a *API) Login(c *gin.Context, r *api.LoginRequest) (*api.LoginResponse, er
 			// this means an external request failed, probably to an IDP
 			return nil, err
 		}
-		logging.S.Debug(err)
```

Comment:
Removed this so that we log all `ErrUnauthorized` in a consistent way. At `Warn` level, from `sendAPIError` with relevant request details.